app/plugins/adp-backend/src/service/armsLengthBodyRouter.ts

```diff
@@ -103,35 +97,37 @@ export async function createRouter(
     );
   }
 
-
   const router = Router();
   router.use(express.json());
- 
-  // Define routes
+  
   router.get('/health', (_, response) => {
```

Comment:
Is this needed? If so, can we improve it to do actual health checks
@jamescoxheadcapgemini  @jeevankuduvaravindran  @RozerinBaran

Reply:
@jamescoxheadcapgemini @kbabigumira What would be the best practice for me to proceed with this?